doc/Language/math.pod6

```diff
@@ -82,7 +82,9 @@ mathematical notation.
 =head1 Arithmetic
 
 Perl 6 can do arithmetic using different data types. L<Num>, L<Rat> and
-L<Complex> can all operate as a L<field under the operations of addition, subtraction, multiplication and division|https://en.wikipedia.org/wiki/Field_(mathematics)>.
+L<Complex> can all operate as a L<field under the operations of addition, subtraction, multiplication and division|https://en.wikipedia.org/wiki/Field_(mathematics)>
+(technically, it should be noted that data types dealing with floating point number representations are not a field in the mathematical sense due to the inherent
+imprecissions of their arithmetic. However, they constitute an approximate enough, computer friendly version of such mathematical objects for most of the cases).
```

Comment:
`imprecissions` → `imprecisions`?

Reply:
Oops, typo. Let me fix that.

Reply:
By the way, if you run `make xtest` it will run some extra tests (one of the test files will do spell checking).